bin/lint-migrations-file/src/change_set/strict.clj

```diff
@@ -122,30 +122,6 @@
    (some change-types-supporting-rollback (mapcat keys changes))
    (contains? change-set :rollback)))
 
-(def ^:private change-types-requiring-preconditions
-  #{:createTable
-    :dropTable
-    :addColumn
-    :dropColumn
-    :createIndex
-    :dropIndex
-    :addForeignKeyConstraint
-    :dropForeignKeyConstraint})
-
-(defn- precondition-present-when-required?
```

Comment:
Removing due to discussion here https://metaboat.slack.com/archives/CKZEMT1MJ/p1748371723075419?thread_ts=1746139506.827869&cid=CKZEMT1MJ